src/plan/generational/immix/global.rs

```diff
@@ -249,10 +249,8 @@ impl<VM: VMBinding> GenImmix<VM> {
         let immix_space = ImmixSpace::new(
             plan_args.get_space_args("immix_mature", true, false, VMRequest::discontiguous()),
             ImmixSpaceArgs {
-                reset_log_bit_in_major_gc: false,
-                // We don't need to unlog objects at tracing. Instead, we unlog objects at copying.
-                // Any object is moved into the mature space, or is copied inside the mature space. We will unlog it.
-                unlog_object_when_traced: false,
+                // We need to unlog objects at tracing time since we currently clear all log bits during a major GC
```

Comment:
Actually we don't clear all log bits during a major GC.  In the current code we set unlog bits during tracing, and clear unlog bits only for freed lines.

Reply:
Eh? This is not true. In the `ImmixSpace::prepare` we do a bulk clear of all the log bits. We do not clear the log bits in the `Block::sweep`.

Reply:
Oops.  My bad.  I forgot you reverted that change.

Reply:
I mean technically, clearing bits lines may be more efficient. But this can be fixed in the internal metadata API refactor.